supported.go

```diff
@@ -0,0 +1,63 @@
+package seccomp
+
+import (
+	"bufio"
+	"errors"
+	"os"
+	"strings"
+
+	perrors "github.com/pkg/errors"
+	"golang.org/x/sys/unix"
+)
+
+const statusFilePath = "/proc/self/status"
+
+// IsSupported returns true if the system has been configured to support
+// seccomp.
+func IsSupported() bool {
+
+	// try to read from status for kernels > 3.8
+	status, err := parseStatusFile(statusFilePath)
+	if err == nil {
+		_, ok := status["Seccomp"]
+		return ok
+	}
+
+	// check if Seccomp is supported via CONFIG_SECCOMP
+	if err := unix.Prctl(unix.PR_GET_SECCOMP, 0, 0, 0, 0); !errors.Is(err, unix.EINVAL) {
```

Comment:
I wonder if this check first is faster?

Reply:
Probably, let's turn them around.

Reply:
Removed the change as discussion in runc: https://github.com/opencontainers/runc/pull/2555#pullrequestreview-468870158